### ISSUE TITLE
Alternative text labels for screen-reader support

### ISSUE THREAD

**Issue by schulle4u** [segler-alex/radiobrowser#124]
When using a screen-reader such as Windows narrator or [NVDA](https://nvaccess.org) to access the site, the buttons and links for each station listing are not accessible. They can be clicked, but due to missing text label the user is not able to fully determine which button they click. Tested also on mobile platforms using TalkBack or Voice Over. I suggest to add some alternative texts to enable blind computer users to use the site. Unfortunately I don't know much about JS stuff, otherwise I would have done this myself. :)
